devito/checkpointing/checkpoint.py

```diff
@@ -72,7 +72,7 @@ def get_data_location(self, timestep):
     @property
     def size(self):
         """The memory consumption of the data contained in a checkpoint."""
-        return sum([int((o.size_allocated/(o.time_order+1))*o.time_order)
+        return sum([int((o.size_allocated/(o.time_size))*o.time_order)
```

Comment:
why does this property need to become public? can/should you not just use `_time_size`? can it be useful on the user side ? (@navjotk )

Reply:
I believe this is the "get" version of the property that the user sets with `save=...`. I think it is useful for the user to be able to read the value if they are allowed to set it - especially since we accept defaults. 

Having said that, I don't feel too strongly about it so happy with it being public or private.